docs/markdown/snippets/find-override.md

```diff
@@ -0,0 +1,27 @@
+## Can override find_program
+
+It is now possible to override the result of `find_program` to point
+to a custom program you want. The overriding is global and applies to
+every subproject from there on. Here is how you would use it.
+
+In master project
+
+```meson
+subproject('mydep')
+```
+
+In the called subproject:
+
+```meson
+prog = find_program('my_custom_script')
+meson.override_find_program('mycodegen', prog)
```

Comment:
This needs an example for `configure_file()` too.

Reply:
It should probably go to actual documentation, people on average don't read old release notes for stuff. That would require writing a proper page for that, though...

Reply:
Maybe we should start linking to the release notes for each feature; the `(added in v0.xx.y)` text is a good candidate for that.